examples/simultaneous-cursors/README.md

```diff
@@ -0,0 +1,61 @@
+# Yorkie Quill Example
```

Comment:
Change title to `Yorkie Simultaneous Cursors Example`

Reply:
thank you for your review! i just fixed it